src/app/util/Mobile/isMobile.js

```diff
@@ -14,9 +14,11 @@ export const isMobile = {
     blackBerry: () => navigator.userAgent.match(/BlackBerry/i),
     iOS: () => navigator.userAgent.match(/iPhone|iPod/i),
     opera: () => navigator.userAgent.match(/Opera Mini/i),
+    vanillaOpera: () => navigator.userAgent.match(/OPR/i),
+    firefox: () => navigator.userAgent.match(/Firefox/i),
     windows: () => navigator.userAgent.match(/IEMobile/i),
     // eslint-disable-next-line max-len
-    any: () => (isMobile.android() || isMobile.blackBerry() || isMobile.iOS() || isMobile.opera() || isMobile.windows()),
+    any: () => (isMobile.android() || isMobile.blackBerry() || isMobile.iOS() || isMobile.opera() || isMobile.windows() || isMobile.vanillaOpera() || isMobile.firefox()),
```

Comment:
This will actually break mobile check on opera and safari. Please don't add this here, keep only `vanillaOpera` and `firefox` in the list.